gen/definitions/router_isis_address_family.yaml

```diff
@@ -105,3 +105,21 @@ attributes:
         example: 1
       - yang_name: maximum-prefixes
         example: 1000
+
+  - yang_name: redistribute/isis
+    tf_name: redistribute_id
+    type: List
+    attributes:
+      - yang_name: instance-id
+        id: true
+        example: CORE
+      - yang_name: route-policy
+        example: ROUTE_POLICY_1
+
+
+# test_prerequisites:
```

Comment:
If it is not needed, this can be removed.

Reply:
done